app/controllers/users_controller.rb

```diff
@@ -170,38 +170,109 @@ def set_vars_from_curr_user
   
 	def show
 		set_vars_from_curr_user
-		@calories = self.class.calc_calories(@gender, @weight, @height, @age, @exercise, @goal).round(0)
-		@all_recipes = Recipe.find_in_api(@calories, @budget, @time)
-		@daily_recipes = self.class.do_daily_recipes(@all_recipes)
+		if current_user.recipes.blank?
```

Comment:
we need to remember to do a check to see if current_user exists in case a user attempts to route to an endpoint without being signed in

Reply:
Good point. We can get on that soon.